eth/sync.go

```diff
@@ -198,8 +199,15 @@ func (cs *chainSyncer) loop() {
 	defer cs.handler.downloader.Terminate()
 
 	for {
-		if op := cs.nextSyncOp(); op != nil {
-			cs.startSync(op)
+		// Checking whether to sync doesn't need to happen so aggressively,
+		// Adding 10 Millisecond of sleep, because Prime chain which is affected by
+		// this change has a block time of 10 Minutes
+		time.Sleep(10 * time.Millisecond)
```

Comment:
Unless there's a good reason, this should be moved under the prime context check.

Reply:
Like https://github.com/Djadih/go-quai/commit/ef1f6b75f7eb57171e22737b09e42e6867583fa6